common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -8,7 +8,11 @@ dbg() {
 }
 
 err() {
-	echo -e '\033[1;31m'"$@"'\033[0m'
+    printf '\033[1;31m'"$@\n"'\033[0m'
```

Comment:
I would have put the newline after the end-of-color, but it doesn't matter much. :ok: